addons/web/static/src/legacy/scss/form_view.scss

```diff
@@ -73,6 +73,11 @@ $o-form-label-margin-right: 0px;
         min-height: $font-size-base * $line-height-base;
     }
 
+    // Explicitly set elements as full-width
+    .o_form_content_fullwidth {
+        @include o-form-nosheet-negative-margin();
+    }
```

Comment:
Not a big fan of the "negative-margin" style but, well, it will be something to discuss further when revamping the Form View.

Could you add a small note in the commit message about which module will use it initially?

Reply:
> Could you add a small note in the commit message about which module will use it initially?

Done, thanks

> Not a big fan of the "negative-margin" style [...]

Do you mean the visual result or the implementation chosen to achieve it?

Reply:
the implementation (ie. `margin: -16px;`)